website/content/docs/ecs/enterprise.mdx

```diff
@@ -59,7 +59,72 @@ features.
+| Namespaces                        | Yes           | This feature requires Consul Enterprise servers and clients.                                                                                                            |
+| Admin Partitions                  | Yes           | This feature requires Consul Enterprise servers and clients.                                                                                                            |
 | Audit Logging                     | No*           | Audit logging can be enabled on Consul servers that run outside of ECS but is not currently supported on the Consul clients that run inside ECS.                        |
 
+### Admin Partitions and Namespaces
+
+Consul on ECS supports [Admin Partitions](/docs/enterprise/admin-partitions) and [Namespaces](/docs/enterprise/namespaces) when Consul Enterprise servers and clients are used.
+These features require:
+* That ACLs are enabled.
+* The ACL Controller is running in the ECS cluster.
+* Mesh-tasks are using a Consul Enterprise client image.
+
+The ACL Controller automatically manages ACL policies and token provisioning for clients and services on the service mesh.
+It also creates admin partitions and namespaces if they do not already exist.
+
+-> **NOTE:** The ACL Controller does not delete admin partitions or namespaces once they are created.
+
+Each ACL Controller manages a single admin partition. Consul on ECS supports one ACL Controller per ECS cluster;
+therefore, the administrative boundary for admin partitions is one admin partition per ECS cluster.
+
+The following example demonstrates how to configure the ACL Controller to enable admin partitions
+and manage an admin partition named `my-partition`. The `consul_partition` field is optional and if it
+is not provided when `consul_partitions_enabled = true`, will default to the `default` admin partition.
+
+<CodeBlockConfig highlight="6-7">
+
+```hcl
+module "acl_controller" {
+  source = "hashicorp/consul/aws-ecs//modules/acl-controller"
+
+  ...
+
+  consul_partitions_enabled = true
+  consul_partition          = "my-partition"
+}
+```
+
+</CodeBlockConfig>
+
+Services are assigned to admin partitions and namespaces through the use of task tags.
+The `mesh-task` module automatically adds the necessary tags to the task definition.
+If the ACL Controller is configured for admin partitions, services on the mesh will
+always be assigned to an admin partition and namespace. If the `mesh-task` does not define
+the partition it will default to the `default` admin partition. Similarly, if a `mesh-task` does
+not define the namespace it will default to the `default` namespace.
+
+To support admin partitions and namespaces, a `mesh-task` must use a Consul Enterprise image.
+By default the `mesh-task` module uses a Consul OSS image. The module provides
+a `consul_image` field that can be used to set the Consul client image to an Enterprise version.
```

Comment:
```suggestion

```

The enterprise image requirement is mentioned at the top of the page, so I'm thinking we don't need to restate it here. Thoughts?